src/services/FacilityService.ts

```diff
@@ -0,0 +1,14 @@
+const KEYS = {
+    facilities: "facilities",
+  };
+  export function getAllFacilities() {
+    if (localStorage.getItem(KEYS.facilities) == null)
+      localStorage.setItem(KEYS.facilities, JSON.stringify([]));
+    return myPromise.then((value) => value);
+  }
+  
+  const myPromise = new Promise((resolve, reject) => {
```

Comment:
convert this to a general function that receives response of api call and resolves it in 1 sec

Reply:
make a good name